golem.yaml

```diff
@@ -0,0 +1,87 @@
+package:
+  name: golem
+  version: "1.2.0"
+  epoch: 0
+  description: "Golem is an open source durable computing platform that makes it easy to build and deploy highly reliable distributed systems."
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - gcc
+      - openssl-dev
+      - pkgconf
+      - protobuf-dev
+      - protoc
+      - rust
+      - rustup
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/golemcloud/golem
+      tag: v${{package.version}}
+      expected-commit: 51d5cbb1b9fcca357561c32577998ef74ce30f3d
+
+  - name: Configure and build
+    runs: |
+      # rustup install stable
+      # rustup target add wasm32-wasip1
+      # rustup target add wasm32-unknown-unknown
+
+      # # Put our rustup-installed toolchains into $PATH
+      # export PATH="$(dirname "$(rustup which cargo)"):$PATH"
+
+      export PATH=$PATH:$HOME/.cargo/bin
+
+      cargo install --force cargo-make
+
+      cargo make --profile ci build-release
+
+      mkdir -p ${{targets.contextdir}}/usr/bin/
+      mv ./target/release/golem-component-service ${{targets.contextdir}}/usr/bin/golem-component-service
```

Comment:
Main package name is `golem` but it only provides `golem-component-service`, wondering if this should have a subpkg with a `ranges` pipeline, or we can drop a TODO block if other binaries needed in the future.

Reply:
maybe we can do this refactoring later on, atm the only component that we need to build is component-service, that's it.